data/src/audiosample.rs

```diff
@@ -1,26 +1,38 @@
 use std::fmt;
 use std::string::*;
 
+/// Parameters describing the representation of an audio sample 
 #[derive(Debug, Copy, Clone, PartialEq)]
 pub struct Soniton {
+    /// Number of bits in each sample.
     pub bits: u8,
+    /// True if the sample data is in big endian layout, false if little endian.
     pub be: bool,
+    /// For data which isn't a multiple of 8 this is true if there is no padding between samples,
```

Comment:
You might say `multiple of 8 bits`

Reply:
Done, also I noticed a typo in `src/data/pixel.rs` `rappresent` instead of `represent` so I fixed that :smile: